### PR TITLE
Additional Tool Fixes

### PR DESCRIPTION
**Pull request by scbedd**
Easy way to disable automatic startup/shutdown.

This was intended to be part of the mac agent PR fixes originally.

### CODE REVIEW

**Review by mccoyp — APPROVED** [review]

File: eng/common/testproxy/test-proxy-docker.yml [hunk 1]
```diff
@@ -16,4 +16,8 @@ steps:
 
   - pwsh: |
       docker container ls -a
-    displayName: Check running container
\ No newline at end of file
+    displayName: Check running container
+
+  - pwsh: |
+      Write-Host "##vso[task.setvariable variable=PROXY_MANUAL_START]true"
```
Comment (weshaggard) [inline eng/common/testproxy/test-proxy-docker.yml:22]: Where do we consume or override this?

Comment (scbedd) [reply eng/common/testproxy/test-proxy-docker.yml:22]: In each language shim. It's a shortcut to tell the auto-spinup (that needs to be implemented for each language's specific platform) to NOT spin up.

Comment (scbedd) [reply eng/common/testproxy/test-proxy-docker.yml:22]: This is necessary in situations where they develop using `docker`, but need to run tests on `mac`.

`mac` _can_ have `docker` through `brew` (and `virtualbox` backend for docker) installed on it as part of the build pipeline, but MS doesn't have the license necessary to use it there.

Comment (scbedd) [reply eng/common/testproxy/test-proxy-docker.yml:22]: This will be documented in the integration guidelines pr.

**Pull request merged**